packages/ui/v2/core/layouts/SettingsLayout.tsx

```diff
@@ -344,7 +344,7 @@ export default function SettingsLayout({
       SettingsSidebarContainer={
         <div
           className={classNames(
-            "fixed inset-y-0 z-50 m-0 h-screen transform overflow-y-scroll border-gray-100 bg-gray-50 transition duration-200 ease-in-out",
+            "fixed inset-y-0 z-50 m-0 h-screen w-56 transform overflow-x-hidden overflow-y-scroll border-gray-100 bg-gray-50 transition duration-200 ease-in-out",
```

Comment:
Providing explicit width same as the parent ensures that it isn't pushed around due to overflow-y scrollbar. But it adds a horizontal scrollbar which is hidden using the `overflow-x-hidden` class